vocabularies/voc4cat/0007013.ttl

```diff
@@ -0,0 +1,14 @@
+PREFIX dcterms: <http://purl.org/dc/terms/>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+
+<https://w3id.org/nfdi4cat/voc4cat_0007013>
+    a skos:Concept ;
+    dcterms:identifier "voc4cat_0007013"^^xsd:token ;
+    skos:broader <https://w3id.org/nfdi4cat/voc4cat_0005007> ;
+    skos:definition "The temperature in the reactor during a catalytic conversion experiment."@en ;
```

Comment:
Where is the temperature exactly measured? A reactor is often not isothermal. Also temperature gradients between reactor and reactant phase and between bulk reactant phase and catalyst surface may exist.
For me the reaction temperature for a  reaction catalyzed by a heterogeneous catalyst is given by the temperature of the surface where the reaction takes place.

Reply:
good question, this probably needs to be specified. Your definition is most likely different values than what is measured and reported in most cases. Maybe we need to add different definitions for setting temperature, measured temperature and "true" reaction temperature, which also will be a distribution over the volume of the reactor, depending on its size, and thermal properties of the reaction

Reply:
@dalito can we leave it like this and then add children with more specified attributes? Alternatively I could add `The **recorded** temperature in the reactor during a catalytic conversion experiment`

Reply:
I am fine with adding more detailed sub-concepts later. The difficulty arises from mixing contexts: reaction vs. reactor. 

For the concept "reaction temperature" I suggest `The temperature at which a chemical reaction is happening.` to which we may add `If the reaction is carried out in an isothermal reactor the reaction temperature is the same as the reactor temperature`. - Though I am not sure if @nmoust likes these kind of additions.

Reply:
Thanks, I am adding reactor temperature now as well, and put the addition here.

Reply:
I have no objection to that. I think that it clarifies the definition even more.

Reply:
I have just realised that there exists already an `experiment temperature`, however the definition refers to a photocatalytic reaction. instead of adding `reactor temperature` we could alternatively make the definition of experiment temperature more inclusive by dropping the 'photo' in the definition. What do you think @nmoust @dalito ?

Reply:
In my update that I just pushed I kept reactor temperature.